src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -222,4 +227,14 @@ private function writeResult(OutputInterface $output)
             '',
         ));
     }
+
+    private function generateSalt(OutputInterface $output)
```

Comment:
this needs to be public (but marked as ``@internal``) to be compatible with PHP 5.3 as you call it from a closure.